tests/components/CMakeLists.txt

```diff
@@ -5,6 +5,7 @@ set(TEST_SOURCES
     test_component_mesh_info.cpp
     test_component_output.cpp
 	test_component_isotropic_remeshing.cpp
```

Comment:
I guess this misalignment is not your fault but please do me the favor and clean it up 🙂